data/avro-schema/src/main/resources/avro/net/corda/data/flow/state/checkpoint/SavedOutput.avsc

```diff
@@ -0,0 +1,28 @@
+{
+  "type": "record",
+  "name": "SavedOutput",
+  "namespace": "net.corda.data.flow.state.checkpoint",
+  "doc": "Record the details of output events to be replayed if necessary.",
+  "fields": [
+    {
+      "name": "topic",
+      "type": "string",
+      "doc": "The topic the event should be sent to."
```

Comment:
This is needed for external messaging which sets the topic in the flow engine, which is then read from the MessageRouter